Sources/SimilaritySearchKit/Core/Index/SimilarityIndex.swift

```diff
@@ -319,15 +333,13 @@ extension SimilarityIndex {
 
         let savedVectorStore = try vectorStore.saveIndex(items: indexItems, to: basePath, as: indexName)
 
-        print("Saved \(indexItems.count) index items to \(savedVectorStore.absoluteString)")
```

Comment:
It would be useful to keep this log, perhaps OSLog with debug log type would be preferred?

Reply:
Good point, will fix it. It was a bit annoying as I was writing many "IndexItems" to disk.